controllers/flowlogspipeline/flp_test.go

```diff
@@ -98,29 +99,57 @@ func getConfig() flowslatest.FlowCollectorSpec {
 				Duration: conntrackTerminatingTimeout,
 			},
 		},
-		Loki: flowslatest.FlowCollectorLoki{
-			Enable: pointer.Bool(true),
-			URL:    "http://loki:3100/",
-			BatchWait: &metav1.Duration{
-				Duration: 1,
-			},
-			BatchSize: 102400,
-			MinBackoff: &metav1.Duration{
-				Duration: 1,
-			},
-			MaxBackoff: &metav1.Duration{
-				Duration: 300,
-			},
-			MaxRetries:   pointer.Int32(10),
-			StaticLabels: map[string]string{"app": "netobserv-flowcollector"},
-		},
+		Loki: getLoki(lokiMode...),
 		Kafka: flowslatest.FlowCollectorKafka{
 			Address: "kafka",
 			Topic:   "flp",
 		},
 	}
 }
 
+func getLoki(lokiMode ...string) flowslatest.FlowCollectorLoki {
```

Comment:
this looks a bit weird to me to allow variable args, while only the 0 index is used, for switching between modes . Is there a reason to do that instead of an explicit switch / single arg?
(anyway this can be addressed in a follow-up)

Reply:
the goal was to have the ability to call `getConfig` with some optionnal param(s) like `getConfig("LOKISTACK")` and keep the default empty `getConfig()`

Can you please add any thoughts on testing in [NETOBSERV-1302](https://issues.redhat.com/browse/NETOBSERV-1302) so we can improve all at once ?